### PR TITLE
[VDO-5739] Add dmtests to nightly

### PR DESCRIPTION
**Pull request by lorelei-sakai**
This adds a single suite to VDO nightly to run the DMTest nightly suite. This is currently contains a single test named "Full". As dmtests are added, the nightly suite definition can be adjusted as appropriate.

### CODE REVIEW

**Review by bjohnsto — APPROVED** [review]
Looks good

File: src/perl/nightly/NightlyRunRules.pm [hunk 1]
```diff
@@ -135,57 +127,25 @@ sub _runPerlTests {
...
-sub _runUDSTestSuite {
-  my ($properties, $testName, $osClass, $suite, $otherArgs)
-    = assertMinMaxArgs(['', ''], 3, 5, @_);
+sub _runTestSuite {
+  my ($properties, $testName, $type, $osClass, $suite, $otherArgs)
+    = assertMinMaxArgs(['', ''], 4, 6, @_);
 
   my $logDir = "$properties->{runLogDir}/$testName/";
-  my $rundir = "$properties->{perlRoot}/udstest";
+  my $rundir = "$properties->{perlRoot}/" . $type . "test";
```
Comment (raeburn) [inline src/perl/nightly/NightlyRunRules.pm:145]: wouldn't /${type}test be simpler than mixing string interpolation and concatenation?

Comment (lorelei-sakai) [reply src/perl/nightly/NightlyRunRules.pm:145]: I wasn't sure that interpolation would work correctly. It's plausibly simpler to read, though.

Comment (lorelei-sakai) [reply src/perl/nightly/NightlyRunRules.pm]: This seems to work okay so I've updated the change to use interpolation instead.

**Review by raeburn — APPROVED** [review]

**Pull request merged**